modules/core/rapid_context.js

```diff
@@ -1,34 +1,89 @@
 import { geoExtent } from '../geo';
 
 import toGeoJSON from '@mapbox/togeojson';
+import { dispatch as d3_dispatch } from 'd3-dispatch'; 
+import { utilRebind } from '../util';
 
-
-export function coreRapidContext(context) {
+export function coreRapidContext() {
     var rapidContext = {};
     rapidContext.version = '1.0.1';
+    var _isRect = undefined; 
```

Comment:
maybe call this `_isTaskBoundsRect` since it's a class-level variable (i.e. not just used locally within `setTaskExtentByGpxData`)?